### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by Tomeccx** [Kieaer/Essentials#105]
**How to reproduce**
1. Install plugin

**Logs**
```
[11-02-2022 15:39:46] [I] [Essentials] Loading
[11-02-2022 15:39:47] [E] Failed to load mod file config/mods/Essentials.jar. Skipping.
[11-02-2022 15:39:47] [E] java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:79)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
        at mindustry.mod.Mods.loadMod(Mods.java:707)
        at mindustry.mod.Mods.loadMod(Mods.java:620)
        at mindustry.mod.Mods.load(Mods.java:275)
        at mindustry.Vars.init(Vars.java:288)
        at mindustry.server.ServerLauncher.init(ServerLauncher.java:57)
        at arc.backend.headless.HeadlessApplication.mainLoop(HeadlessApplication.java:66)
        at arc.backend.headless.HeadlessApplication$1.run(HeadlessApplication.java:54)
Caused by: java.lang.NoClassDefFoundError: mindustry/game/EventType$MenuOptionChooseEvent
        at essentials.Event.register(Event.kt:284)
        at essentials.Main.<init>(Main.kt:38)
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
        ... 9 more
Caused by: java.lang.ClassNotFoundException: mindustry.game.EventType$MenuOptionChooseEvent
        at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        ... 12 more

```

**Additional context**

Plugin browser / MTDDirtyWordMask. I have java 19

**Comment by Kieaer** [comment]
The current plugin version is available in Mindustry 7.0 beta build.
Check your mindustry server version.

**Comment by Kieaer** [comment]
Resolved due to release of Mindustry 7.0.
